src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/DefaultRazorProjectService.cs

```diff
@@ -209,7 +209,7 @@ public override void UpdateDocument(string filePath, SourceText sourceText, int
         TrackDocumentVersion(textDocumentPath, version);
     }
 
-    public override void AddProject(string filePath)
+    public override void AddProject(string filePath, string? rootNamespace = null)
```

Comment:
Not a big fan of optional parameters. Does it cause mountains of additional changes to force it to be passed in? Or maybe add a non-abstract overload to the base class?